Dockerfile

```diff
@@ -34,17 +35,14 @@ RUN docker-php-ext-install \
     zip \
     calendar
 
-RUN docker-php-ext-configure \
-    calendar
-
 COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
 
 ENV PHP_MEMORY_LIMIT=512M
 
 RUN cd /usr/local/etc/php/conf.d/ && \
   echo 'memory_limit = 512M' >> /usr/local/etc/php/conf.d/docker-php-memory-limit.ini
 
-ENV NODE_VERSION=12.18.3
+ENV NODE_VERSION=16.15.1
```

Comment:
```suggestion
ENV NODE_VERSION=lts/gallium
```

Reply:
@thomasowow I updated it to `16.17.0` but I couldn't use lts/gallium, because it's not valid for other commands except nvm install